### DISCUSSION TITLE
v0.9.3

### DISCUSSION THREAD

**Discussion by EryouHao** [getgridea/gridea#987] [General]
## ⚠️ 重要更新
**调整 output 文件夹目录位置到 `～/.gridea/output` 文件夹，与站点源文件夹分离，站点源文件夹中的 `output` 文件夹可自行删除。v0.9.3 及后续版本将不再使用。**

额外带来的变更为 Git 提交记录仅保留最新同步的一条。（**如果对提交记录有需求的用户请谨慎升级**。）

![commit](https://user-images.githubusercontent.com/17328747/168728508-df501c1e-2f8b-444b-a81e-fa1f0563bede.jpg)

- 为什么要变更 `output` 目录？
   目的是为了和博客源文件夹分离开，避免使用 OneDrive 等同步工具同步时，把 output 文件夹以及里面的 .git 文件夹一并同步，从而导致遇到同步问题或大量文件变更情况。

## 新增
- 🌟 Github Pages 远程同步**支持配置本地网络代理**。感谢 @wherelse 
![http-proxy](https://user-images.githubusercontent.com/17328747/168728539-edf056c8-9c79-4e99-a825-22c6e049916d.jpg)

- **支持部署到 Netlify**。
![netlify-deploy](https://user-images.githubusercontent.com/17328747/168728566-672cf086-e6ee-4ff6-ace2-845f1ca38bec.jpg)

- 支持部署到 Gitee。感谢 @jiangrubin 

## 修复
- 修复预览时找不到 localhost 时崩溃问题。感谢 @CallanBi 

## 优化
- 调整部分 UI 样式。

<hr /><em>This discussion was created from the release <a href='https://github.com/getgridea/gridea/releases/tag/v0.9.3'>v0.9.3</a>.</em>

**Comment by microhardsmith** [comment]
能支持m1吗

**Comment by cos2202** [comment]
Ubuntu 22.04
Appimage 闪退
Snap 提示不支持

**Comment by unmaned-air** [comment]
下载之后原文件全部消失呜呜呜

**Comment by rakuyoMo** [comment]
“Git 提交记录仅保留最新同步的一条” 还是希望这个功能能有个开关来控制。

**Comment by YoloBird** [comment]
gitee该怎么配置，死活配置不上
